core/src/test/java/io/questdb/test/cutlass/http/ImportIODispatcherTest.java

```diff
@@ -757,6 +757,65 @@ public void testImportWitNocacheSymbolsLoop() throws Exception {
         }
     }
 
+    @Test
+    public void testImportWithCreateTableFalseAndNoTable() throws Exception {
+        new HttpQueryTestBuilder()
+                .withTempFolder(root).withWorkerCount(2)
+                .withHttpServerConfigBuilder(new HttpServerConfigurationBuilder())
+                .withTelemetry(false)
+                .run((engine) -> {
+                    String request = ValidImportRequest1
+                            .replace("POST /upload?name=trips HTTP", "POST /upload?name=trips&timestamp=Pickup_DateTime&createTable=false HTTP");
+
+                    String response = WarningValidImportResponse1
+                            .replace(
```

Comment:
nit: it may be cleaner to introduce a new constant instead of doing `WarningValidImportResponse1.replace` multiple times.

Reply:
Yeah, a new constant will make the code cleaner, so it's worth introducing it.